### PR TITLE
LPD-18346 The order of the items does not matter. We must check if the array contains the expected values

### PR DESCRIPTION
**Pull request by marcelabc**
Related issue: https://liferay.atlassian.net/browse/LPD-18346

### CODE REVIEW

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by marcelabc** [comment]
ci:test:object

File: modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java [hunk 1]
```diff
@@ -4998,12 +4998,20 @@ public void testPostCustomObjectEntryWithNestedCustomObjectEntriesInManyToManyRe
...
+				).put(
+					"externalReferenceCode", _ERC_VALUE_1
+				),
+				JSONUtil.put(
+					_OBJECT_FIELD_NAME_2, _NEW_OBJECT_FIELD_VALUE_2
+				).put(
+					"externalReferenceCode", _ERC_VALUE_2
+				)
+			).toString(),
+			nestedObjectEntriesJSONArray.toString(), JSONCompareMode.LENIENT);
```
Comment (carolmariaabb) [inline modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java:5014]: @marcelabc it's always the same code, can we move it to a private method?

```
private void _jsonaAssertEquals(JSONArray nestedObjectEntriesJSONArray) {
	JSONUtil.putAll(
		JSONUtil.put(
			_OBJECT_FIELD_NAME_2, _NEW_OBJECT_FIELD_VALUE_1
		).put(
			"externalReferenceCode", _ERC_VALUE_1
		),
		JSONUtil.put(
			_OBJECT_FIELD_NAME_2, _NEW_OBJECT_FIELD_VALUE_2
		).put(
			"externalReferenceCode", _ERC_VALUE_2
		)
	).toString(),
	nestedObjectEntriesJSONArray.toString(), JSONCompareMode.LENIENT);
}
```

**Review by carolmariaabb — CHANGES_REQUESTED** [review]

**Comment by marcelabc** [comment]
ci:test:sf

**Review by carolmariaabb — APPROVED** [review]

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**